detector/aws_db_instance_default_parameter_group.go

```diff
@@ -10,20 +10,17 @@ import (
 
 type AwsDBInstanceDefaultParameterGroupDetector struct {
 	*Detector
-	IssueType  string
-	TargetType string
-	Target     string
-	DeepCheck  bool
 }
 
 func (d *Detector) CreateAwsDBInstanceDefaultParameterGroupDetector() *AwsDBInstanceDefaultParameterGroupDetector {
-	return &AwsDBInstanceDefaultParameterGroupDetector{
-		Detector:   d,
-		IssueType:  issue.NOTICE,
-		TargetType: "resource",
-		Target:     "aws_db_instance",
-		DeepCheck:  false,
-	}
+	nd := &AwsDBInstanceDefaultParameterGroupDetector{Detector: d}
+	nd.Name = "aws_db_instance_default_parameter_group"
+	nd.IssueType = issue.NOTICE
+	nd.TargetType = "resource"
+	nd.Target = "aws_db_instance"
```

Comment:
__[gometalinter]__
* [goconst] 8 other occurrence(s) of "aws_db_instance" found in: aws_db_instance_invalid_type.go:23:14 aws_db_instance_readable_password.go:17:14 aws_db_instance_invalid_option_group.go:23:14 aws_db_instance_invalid_vpc_security_group.go:24:14 aws_db_instance_invalid_parameter_group.go:23:14 aws_db_instance_previous_type.go:23:14 aws_db_instance_duplicate_identifier.go:23:14 aws_db_instance_invalid_db_subnet_group.go:23:14

<!-- Comment by [SideCI](https://sideci.com) -->